ImageGallery/ImageGallery/Model/PexelPhoto.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  PexelPhoto.swift
+//  ImagaeGallery
+//
+//  Created by Nick Ryan on 6/18/24.
+//
+
+import Foundation
+
+struct PexelsPhoto: Codable, Identifiable {
+  let id: Int
+  let width: Int
+  let height: Int
+  let url: String
+  let photographer: String
+  let photographerURL: String
+  let src: PhotoSrc
+  
+  private enum CodingKeys: String, CodingKey {
+    case id, width, height, url, photographer
+    case photographerURL = "photographer_url"
+    case src
+  }
+  
+  struct PhotoSrc: Codable {
+    let original: String
+    let large: String
+    let large2x: String
```

Comment:
These are URLs right? Instead of `String` here, you can use the type `URL` and Coddle will transform the url it gets in the response into an `URL` object.